Ships as Raw

```diff
@@ -0,0 +1,860 @@
+
+Name|Gladiator I-class Star Destroyer
+Type|Gladiator
+Keywords|0
+Faction|Imperial
+Size|Small
+Points|56
+Hull|5
+defenseTokens|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|3
+SideShields|2
+RearShields|1
+FrontArmament|RRKK
+SideArmament|KKKK
+RearArmament|RK
+AntiSquadron|B
+Upgrade|Title Officers Weapon Team Support Ordance
+FrontArcOffset|8.7
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|7.5
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|33
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|37
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|Gladiator II-class Star Destroyer
+Type|Gladiator
+Keywords|0
+Faction|Imperial 
+Size|Small
+Points|62
+Hull|5
+defenseTokens|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|3
+SideShields|2
+RearShields|1
+FrontArmament|RRKK
+SideArmament|KKKR
+RearArmament|RK
+AntiSquadron|BB
+Upgrade|Title Officers Weapon Team Support Ordance
+FrontArcOffset|8.7
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|7.5
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|33
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|37
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+
+Name|Victory I-class Star Destroyer
+Type|Victory
+Keywords|0
+Faction|Imperial
+Size|Medium
+Points|73
+Hull|8
+defenseTokens|Rd Rd Br
+Command|3
+Squad|3
+Eng|4
+Speed1|1
+Speed2|01
+Speed3|0
+Speed4|0
+FrontShields|3
+SideShields|3
+RearShields|1
+FrontArmament|RRRKKK
+SideArmament|KRRR
+RearArmament|RR
+AntiSquadron|B
+Upgrade|Title Officers TurboLaser Offensive Weapon Team Ordance   
+FrontArcOffset|21
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|21
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|47
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|54
+//How far from the rear of the ship do the rear hull zone lines meet//
+//This ship has a 1 mm section between the rear and front conjuctions.//
+
+Name|Victory II-class Star Destroyer
+Type|Victory
+Keywords|0
+Faction|Imperial
+Size|Medium
+Points|85
+Hull|8
+defenseTokens|Rd Rd Br
+Command|3
+Squad|3
+Eng|4
+Speed1|1
+Speed2|01
+Speed3|0
+Speed4|0
+FrontShields|3
+SideShields|3
+RearShields|1
+FrontArmament|RRRBBB
+SideArmament|BRR
+RearArmament|RR
+AntiSquadron|B
+Upgrade|Title Officers TurboLaser Offensive Weapon Team Ion cannon    

```

Comment:
Turbolaser, not TurboLaser.  I *think* its one word, but my lack of SW knowledge might be failing me